scripts/rackhd.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+import json
+import requests
+import os
+import argparse
+import types
+
+RACKHD_URL = 'http://localhost:8080'
```

Comment:
We might want to render this port as a template variable given that it's configurable by the user.